packs/other-effects/effect-aquatic-combat.json

```diff
@@ -103,6 +103,47 @@
                     }
                 ],
                 "uuid": "Compendium.pf2e.conditionitems.Item.Off-Guard"
+            },
+            {
+                "key": "DamageAlteration",
+                "mode": "override",
+                "predicate": [
+                    {
+                        "or": [
+                            "item:damage:persistent:fire",
+                            "item:damage:fire"
+                        ]
+                    }
+                ],
+                "property": "dice-number",
+                "selectors": [
+                    "inline-damage",
+                    "attack-damage",
+                    "spell-damage"
+                ],
+                "value": 0
+            },
+            {
+                "damageType": "fire",
+                "key": "AdjustModifier",
+                "predicate": [
+                    {
+                        "or": [
+                            "item:damage:persistent:fire",
+                            "item:damage:fire"
+                        ]
+                    }
+                ],
+                "selectors": [
+                    "inline-damage",
+                    "attack-damage",
+                    "spell-damage"
+                ],
+                "suppress": true
+            },
+            {
+                "key": "GrantItem",
+                "uuid": "Compendium.pf2e.other-effects.Item.Effect: Remaining Air"
             }
```

Comment:
I think maybe just link to it in the description. Granting isn't needed for actors that can breath underwater.